python/rapidsmpf/rapidsmpf/integrations/dask/shuffler.py

```diff
@@ -280,13 +314,17 @@ def _extract_partition(
     -------
     Extracted DataFrame partition.
     """
-    if callback is None:
-        raise ValueError("Missing callback in _extract_partition.")
```

Comment:
Looks like this check is not needed anymore because the callback that is passed is `DaskIntegration.extract_partition`, and for an object to qualify for the `DaskIntegration` the `extract_partition` must exist properly implemented as the method in the protocol, is that right?

Reply:
Exactly